rllib/core/rl_trainer/tf/tf_rl_trainer.py

```diff
@@ -190,7 +259,10 @@ def convert_batch_to_tf_tensor(self, batch: MultiAgentBatch) -> NestedDict:
         # tf.function. This messes with input spec checking. Other fields of
         # the sample batch are possibly modified by tf.function which may lead
         # to unwanted consequences. We'll need to further investigate this.
-        batch = NestedDict(batch.policy_batches)
-        for key, value in batch.items():
-            batch[key] = tf.convert_to_tensor(value, dtype=tf.float32)
+        batch = batch.policy_batches
```

Comment:
I needed to do this because when I call rl_optimizer.compute_loss(batch), the function signature expects a mapping, but really it expects a dictionary.